docs-mk/fr/docs/guide/messenger.md

```diff
@@ -26,8 +26,14 @@ Si vous le souhaitez, vous pouvez aussi sauter cette étape et passer directemen
 
 * Créez une page Facebook
 
+![Créer une page partie 1](https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-page-0.png)
+
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-page-1.png" alt="Créer une page partie 2" style="zoom:50%;" />
```

Comment:
I have the impression that the zoom:50% does not work - the second image in the page (and others) are too big compared to others.